packages/app-desktop/gui/Sidebar/Sidebar.tsx

```diff
@@ -448,7 +448,7 @@ class SidebarComponent extends React.Component<Props, State> {
 
 	renderTag(tag: any, selected: boolean) {
 		const anchorRef = this.anchorItemRef('tag', tag.id);
-		const noteCount = Setting.value('showNoteCounts') ? this.renderNoteCount(tag.note_count) : '';
+		const noteCount = Setting.value('showNoteCounts') ? Setting.value('showCompletedTodos') ? this.renderNoteCount(tag.note_count) : this.renderNoteCount(tag.note_count - tag.todo_completed_count) : '';
```

Comment:
Could you refactor this logic to avoid the nested ternary operators? This is to clarify a bit the logic.